src/features/lecture-evaluation/components/LectureEvaluationIntro/LectureEvaluationIntro.module.scss

```
@@ -0,0 +1,527 @@
+.disabledTag {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 45px;
+  padding: spacing.$space-8;
+  color: colors.$gray-200;
+  border: 1px solid colors.$gray-300;
+  border-radius: 6px;
+  @include typography.body-sm-design;
+  @include typography.font-weight-extrabold;
+  text-align: center;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)